kube-client/src/api/subresource.rs

```diff
@@ -248,8 +247,9 @@ where
         self.client.request_text(req).await
     }
 
-    /// Fetch logs as a stream of bytes
-    pub async fn log_stream(&self, name: &str, lp: &LogParams) -> Result<impl Stream<Item = Result<Bytes>>> {
+    /// Fetch logs in the form of an async buf reader. Users can call `.lines()`
```

Comment:
Would be nice if the lines sentence was a code example:

    # Example
    
    ```no_run
    <code here>
    ```

We have a bunch of those in the controller module.